### DISCUSSION TITLE
.net 4.8 System.TypeInitializationException - Unable to load DLL 'Magick.Native-Q16-x64.dll'

### DISCUSSION THREAD

**Discussion by dtechfish** [dlemstra/Magick.NET#1292] [Help] [answered]
Just looking for a push in the right direction....

Running simple image resize operations and format conversions that work fine in other MVC or Core Console apps... but I can't seemt to get it wire up in a refactored asp.net app targeting framework 4.8.

Getting exception in MagickSettings.cs at line:
using (var instance = new NativeMagickSettings())

_Exception Thrown:_
System.TypeInitializationException: 'The type initializer for 'NativeMagickSettings' threw an exception.'

Inner Exception:
DllNotFoundException: Unable to load DLL 'Magick.Native-Q16-x64.dll': The specified module could not be found. 

Installed NuGet Packages:
Magick.Net-Q16-x64 also tried AnyCpu v12.2.0
Magick.Net.Core v12.2.0
NETStandard.Library 2.0.3

I do not see the Magick.Native dlls getting copied to the bin... I don't remember doing this in the past and I can't seem to find them anywhere.  Is this the culprit or is there something else I'm missing?

Thanks in advance.

**Comment by dlemstra** [comment, marked as answer]
I don't know why the native binaries are not copied but you can force the copy with the following settings: https://github.com/dlemstra/Magick.NET/blob/main/docs/CrossPlatform.md#native-library-is-not-copied-to-the-bin-folder

**Reply by dtechfish** [reply]
Thank you worked like a charm.
